### PR TITLE
Manage institution mobile page

### PR DESCRIPTION
**Pull request by ruanres**
**Feature/Bug description:** 

**models:**

invite-inst-model
parent-model
children-model
pendent-links-model

**Solution:** 
invite-inst
parent-inst
children-inst
vinculos-notificacao

**TODO/FIXME:** n/a

### CODE REVIEW

**Review by tiago-lp — APPROVED** [review]

File: frontend/institution/manageInstitution/management_institution_mobile.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+        <clickable-card icon="account_balance" title="Hierarquia Superior">
+            <div class="manage-institution__card__content">
+                <entity-showcase
+                    ng-if="inviteInstHierCtrl.hasParent"
+                    icon="account_balance"
+                    title="inviteInstHierCtrl.institution.parent_institution.name"
+                    subtitle="inviteInstHierCtrl.getStatusMsg(inviteInstHierCtrl.institution.parent_institution, true)"
+                    right-icon-btns="[
+                        inviteInstHierCtrl.createIconBtn('more_vert',
```
Comment (brobsc) [inline frontend/institution/manageInstitution/management_institution_mobile.html]: Change remove link icon on superior institutions

**Review by brobsc — CHANGES_REQUESTED** [review]

File: frontend/institution/manageInstitution/management_institution_mobile.html [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                </div>
+                <load-circle ng-if="manageMemberCtrl.isLoadingInvite"></load-circle>
+            </div>
+        </clickable-card>
+
+        <clickable-card icon="account_balance" title="Hierarquia Superior">
+            <div class="manage-institution__card__content">
+                <entity-showcase
+                    ng-if="inviteInstHierCtrl.hasParent"
+                    icon="account_balance"
```
Comment (JuliePessoa) [inline frontend/institution/manageInstitution/management_institution_mobile.html:48]: icon onclick should go to institution page

File: frontend/institution/manageInstitution/management_institution_mobile.html [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+            </div>
+        </clickable-card>
+
+        <clickable-card icon="account_balance" title="Hierarquia Subordinada">
+            <div class="manage-institution__card__content">
+                <search-field keyword="searchSubordinate"></search-field>
+                <md-content class="manage-institution__list custom-scrollbar">
+                    <entity-showcase 
+                        ng-repeat="subInst in inviteInstHierCtrl.institution.children_institutions | filter: searchSubordinate" 
+                        icon="account_balance"
```
Comment (JuliePessoa) [inline frontend/institution/manageInstitution/management_institution_mobile.html:66]: icon onclick should go to institution page

**Review by JuliePessoa — CHANGES_REQUESTED** [review]

File: frontend/institution/manageInstitution/management_institution_mobile.html [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+        </clickable-card>
+
+        <clickable-card icon="feedback" title="Vínculos Pendentes">
+            <div class="manage-institution__card__content">
+                <search-field keyword="searchInstRequest"></search-field>
+                <md-content class="manage-institution__list custom-scrollbar">
+                    <entity-showcase 
+                        ng-repeat="instRequest in inviteInstHierCtrl.requested_invites | filter: searchInstRequest"
+                        ng-if="inviteInstHierCtrl.isReqSentToCurrentInst(instRequest)"
+                        icon="account_balance"
```
Comment (JuliePessoa) [inline frontend/institution/manageInstitution/management_institution_mobile.html:86]: icon onclick should go to institution page

**Review by JuliePessoa — CHANGES_REQUESTED** [review]

**Review by JuliePessoa — CHANGES_REQUESTED** [review]
green dot is missing on "Vinculos Pendentes" and "Convites Requisitados".
image

**Review by JuliePessoa — APPROVED** [review]

**Pull request merged**